packages/core/src/components/checkbox/checkbox.tsx

```diff
@@ -39,10 +39,15 @@ export class CheckboxComponent {
   @Prop() value: any | null;
 
   /**
-   * The text that's displayed alongside the checkbox
+   * The label text must be provided and is displayed beside the checkbox, use the labelHidden property to control its visibility.
```

Comment:
If you add back ticks to the labelHidden string in the comment it will format nicely when rendered on the website

Compare to the `value` prop comment text:

<img width="1188" alt="image" src="https://github.com/user-attachments/assets/826c88d1-3331-4928-b8ad-f29e8bf99955">

```suggestion
   * The label text must be provided and is displayed beside the checkbox, use the `labelHidden` property to control its visibility.
```